functions/fuel.js

```diff
@@ -0,0 +1,8 @@
+'use strict';
+var mpg = 30;
+var distance = 204;
+var fuelCalc = function (){
+    var answer = (distance / mpg);
+    console.log("The amount of gallons you need to travel is " + answer);
```

Comment:
Don't output to console - not part of the function's responsibility: it just calculates.